src/Drivers/nlpMDS_raja_ex4.cpp

```diff
@@ -6,18 +6,9 @@
 #include <hiopMatrixDenseRowMajor.hpp>
 #include <hiopMatrixRajaDense.hpp>
 
-#ifdef HIOP_USE_GPU
-  #include "cuda.h"
-  #define RAJA_CUDA_BLOCK_SIZE 128
-  using ex4_raja_exec   = RAJA::cuda_exec<RAJA_CUDA_BLOCK_SIZE>;
-  using ex4_raja_reduce = RAJA::cuda_reduce;
-  #define RAJA_LAMBDA [=] __device__
-#else
-  using ex4_raja_exec   = RAJA::omp_parallel_for_exec;
-  using ex4_raja_reduce = RAJA::omp_reduce;
-  #define RAJA_LAMBDA [=]
-#endif
-
+#include <hiop_raja_defs.hpp>
```

Comment:
Since we include this, why not just replace `ex4_raja_exec` with `hiop::hiop_raja_exec` for the moment?

Reply:
Fair enough. I figured there was a reason, but I wanted to ask the question :)